tests/demo/demo_frame_graphs.hpp

```diff
@@ -204,7 +204,7 @@ namespace fg_manager
 			wr::AddShadowDenoiserTask(*fg);
 
 			//Raytraced Ambient Occlusion task
-			wr::AddRTAOTask(*fg, static_cast<wr::D3D12RenderSystem&>(rs).m_device);
+			//wr::AddRTAOTask(*fg, static_cast<wr::D3D12RenderSystem&>(rs).m_device);
```

Comment:
Remove commented-out code to keep the codebase clean.

Reply:
I don't think this is supposed to be pushed either.

Reply:
👍